tool/dumper/vkgcPipelineDumper.cpp

```diff
@@ -665,6 +665,7 @@ void PipelineDumper::dumpPipelineOptions(const PipelineOptions *options, std::os
   dumpFile << "options.scalarBlockLayout = " << options->scalarBlockLayout << "\n";
   dumpFile << "options.includeIr = " << options->includeIr << "\n";
   dumpFile << "options.robustBufferAccess = " << options->robustBufferAccess << "\n";
+  dumpFile << "options.robustBufferAccess = " << options->enableScratchBoundsCheck << "\n";
```

Comment:
Presumably duplicate "options.robustBufferAccess" should be "options.enableScratchBoundsCheck"?